mongo/client.go

```diff
@@ -193,7 +193,7 @@ func newClient(opts ...*options.ClientOptions) (*Client, error) {
 	}
 	// AutoEncryptionOptions
 	if clientOpts.AutoEncryptionOptions != nil {
-		if err := client.configureAutoEncryption(clientOpts); err != nil {
+		if err = client.configureAutoEncryption(clientOpts); err != nil {
```

Comment:
Was there a particular scoping or style reason for migrating from the variable declaration to assignment?